Week1/homework/app.js

```diff
@@ -1,11 +1,187 @@
-'use strict';
+/* eslint-disable no-unused-expressions */
+{
+  ('use strict');
+  let bookTitles = [
+    'deep_work',
+    'anna_frank',
+    'forty_rules',
+    'memory_in_flesh',
+    'dust_of_promises',
+    'art_of_forgetting',
+    'black_suits_you',
+    'Chaos_Of_Senses',
+    'bed_hopper',
+    'game_of_thrones',
+  ];
+  console.log(bookTitles);
+
+  const listElement = document.createElement('ul');
+  function listElements(ul, itemsArray) {
+    const root = document.getElementById('root');
+    root.appendChild(ul);
+    for (let i = 0; i < itemsArray.length; i++) {
+      const listItem = itemsArray[i];
+      const li = document.createElement('li');
+      li.innerText = listItem;
+      ul.appendChild(li);
+    }
+  }
+  //listElements(listElement, bookTitles);
+}
 
 {
+  ('use strict');
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    'deep_work',
+    'anna_frank',
+    'forty_rules',
+    'memory_in_flesh',
+    'dust_of_promises',
+    'art_of_forgetting',
+    'black_suits_you',
+    'Chaos_Of_Senses',
+    'bed_hopper',
+    'game_of_thrones',
   ];
+  // console.log(bookTitles);
 
-  // Replace with your own code
-  console.log(bookTitles);
+  function elements() {
+    const ul = document.createElement('ul');
+    const root = document.getElementById('root');
+    root.appendChild(ul);
+    for (let i = 0; i < bookTitles.length; i++) {
+      const bookElem = bookTitles[i];
+      const li = document.createElement('li');
+      li.innerText = bookElem;
+      ul.appendChild(li);
+    }
+  }
+  // elements();
+}
+// const load = () => {
+//   console.log('load event detected!');
+//   elements();
+// };
+// window.onload = load;
+{
+  ('use strict');
```

Comment:
You only need to use `('use strict')` once at the top of the file.

Reply:
Good to know!